src/setup/components-global.js

```diff
@@ -1,21 +1,12 @@
-// Form elements
```

Comment:
todo: I would prefer if we could name the file `ssr-components.js` instead. (SSR = Server Side Rendering)